nautilus/include/nautilus/val.hpp

```diff
@@ -160,6 +160,12 @@ class val<ValueType> {
 		*this = *this - (ValueType) 1;
 		return temp;
 	}
+
+	friend std::ostream &operator<<(std::ostream &os, const val<ValueType>& value) {
```

Comment:
This is not working in the generated code but nautilus now supports this in a better fashion using iostream